### ISSUE TITLE
[Guide] Test Gutenberg Block Editor Accessibility

### ISSUE THREAD

**Issue by chekle** [WordPress/contributor-handbook#234]
### Function

Test

### Type

Task: quick start, repeatable work

### Summary

Review GitHub issues labeled "Needs Accessibility Feedback" for the Gutenberg block editor. Test new blocks and features using keyboard navigation, screen readers, and accessibility testing tools. Help close remaining issues from the WPCampus audit and ensure the block editor is accessible to all users.

### Links & Details

**Gutenberg Accessibility:**
- Gutenberg Accessibility Issues: https://github.com/WordPress/gutenberg/labels/Accessibility%20%28a11y%29
- Needs Accessibility Feedback: https://github.com/WordPress/gutenberg/issues?q=is%3Aopen+is%3Aissue+label%3A%22Needs+Accessibility+Feedback%22
- Gutenberg Repository: https://github.com/WordPress/gutenberg

**Testing Resources:**
- Audits and Testing Guide: https://make.wordpress.org/accessibility/handbook/get-involved/audits-and-testing/
- Test for Web Accessibility: https://make.wordpress.org/accessibility/handbook/best-practices/test-for-web-accessibility/
- Make WordPress Accessibility: https://make.wordpress.org/accessibility/

**Community:**
- WordPress Slack #accessibility: https://wordpress.slack.com/archives/accessibility
- Join WordPress Slack: https://make.wordpress.org/chat/
- Gutenberg Working Group: https://make.wordpress.org/accessibility/handbook/get-involved/#gutenberg-working-group

**Comment by chekle** [comment]
V1 pathway doc [here](https://github.com/WordPress/contributor-handbook/blob/main/pathways/test/test-gutenberg-block-editor-accessibility.md)

**Comment by chekle** [comment]
Asking for feedback [here](https://wordpress.slack.com/archives/C02RP4X03/p1781056579502189)

**Comment by chekle** [comment]
Moving this to blocked after receiving feedback. There are other processes that are changing and will affect this pathway, but we can revisit it later.
